rust/worker/src/system/wrapped_message.rs

```diff
@@ -31,25 +70,17 @@ where
 }
 
 #[async_trait]
-impl<C, M> HandleableMessage<C> for Option<M>
+impl<C, M> HandleableMessage<C> for Option<MessageWithReplyChannel<M, C::Result>>
 where
     C: Component + Handler<M>,
     M: Debug + Send + 'static,
 {
     async fn handle(&mut self, component: &mut C, ctx: &ComponentContext<C>) -> () {
```

Comment:
nit - since this is what executor now calls, I am wondering if we should call it handle_and_reply so that the executor line 89 where its called reads clearly

Reply:
makes sense